### ISSUE TITLE
`^` is xor, not exponentiation

### ISSUE THREAD

**Issue by dgryski** [vitessio/vitess#4928]
https://github.com/vitessio/vitess/blob/a92a21ae79bee88cd208907c668d36deb2693dd8/go/vt/topo/memorytopo/memorytopo.go#L175

```
		generation: uint64(rand.Int63n(2 ^ 60)),
```

You probably mean
```
		generation: uint64(rand.Int63n(1 << 60)),
```

**Comment by sougou** [comment]
Nice catch :bowing_man:
@dkhenry

**Comment by sougou** [comment]
Closed via #4950
